### ISSUE TITLE
NoSuchMethodError GeneratedMessageV3.isStringEmpty() when used in maven project

### ISSUE THREAD

**Issue by jenshb** [camunda-community-hub/micronaut-zeebe-client#88]
When micronaut-zeebe-client is used as dependency in a maven project, the following error occurs after connecting to the Zeebe-engine:
```
Exception in thread "grpc-default-executor-0" java.lang.NoSuchMethodError: 'boolean com.google.protobuf.GeneratedMessageV3.isStringEmpty(java.lang.Object)'
	at io.camunda.zeebe.gateway.protocol.GatewayOuterClass$ActivateJobsRequest.getSerializedSize(GatewayOuterClass.java:499)
	at io.grpc.protobuf.lite.ProtoInputStream.available(ProtoInputStream.java:108)
	at io.grpc.internal.MessageFramer.getKnownLength(MessageFramer.java:205)
	at io.grpc.internal.MessageFramer.writePayload(MessageFramer.java:137)
	at io.grpc.internal.AbstractStream.writeMessage(AbstractStream.java:65)
	at io.grpc.internal.ForwardingClientStream.writeMessage(ForwardingClientStream.java:37)
	at io.grpc.internal.DelayedStream$6.run(DelayedStream.java:282)
	at io.grpc.internal.DelayedStream.drainPendingCalls(DelayedStream.java:181)
	at io.grpc.internal.DelayedStream.access$100(DelayedStream.java:43)
	at io.grpc.internal.DelayedStream$4.run(DelayedStream.java:147)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
```

micronaut-zeebe-client version 1.4.0
micronaut version 3.3.4

**Reason:**
In micronaut-bom 3.3.4 version 3.17.2 from com.google.protobuf:protobuf-bom is required. But the method isStringEmpty() was added to GeneratedMessageV3 only in version 3.18, so zeebe-client-java requires at least this version.

The dependency conflict can be solved by adding explicitly a newer version of com.google.protobuf:protobuf-java to the project, e. g. 
```
    <dependency>
      <groupId>com.google.protobuf</groupId>
      <artifactId>protobuf-java</artifactId>
      <version>3.19.4</version>
    </dependency>
```

The problem does not occur when build with Gradle, because Gradle prefers the higher version in a version conflict.

**Comment by jenshb** [comment]
Will be fixed by this Pull-Request in micronaut-grpc:
https://github.com/micronaut-projects/micronaut-grpc/pull/460

**Comment by tobiasschaefer** [comment]
@jenshb , @slackwing:

I've just published release 1.8.0 of micronaut-camunda-zeebe which is built with Micronaut 3.5.2. With this release the correct version of protobuf-java is being used.

Note: a similiar problem like this can unfortunately occur again in the future if Zeebe is updated and requires a newer version than is provided by Micronaut.
